modules/auxiliary/sqli/openemr/openemr_sqli_dump.rb

```diff
@@ -83,113 +84,9 @@ def get_response(payload)
         'pid' => '1',
         'doc_type' => '1',
         'doc_id' => '1',
-        'enc' => "1' and updatexml(1,concat(0x7e, (#{payload})),0) or '"
+        'enc' => "1' and updatexml(1,concat(0x7e, (" + payload + ")),0) or '"
```

Comment:
```suggestion
        'enc' => "1' and updatexml(1,concat(0x7e, (#{payload})),0) or '"
```

Revert to original format please.